ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите 1 катет: ");
```

Comment:
Порядковое числительное:
```suggestion
            Console.WriteLine("Введите 1-й катет: ");
```